app/assets/stylesheets/navbar.css

```diff
@@ -3,12 +3,16 @@
         display: block;
     }
 
+    /*.navbar.not-transparent {*/
+        /*background-color: #0f3157 !important;*/
+    /*}*/
+
     .navbar {
-        background-color: transparent;
+        background-color: #0f3157 !important;
```

Comment:
Can you move this out of the `media` selector because it's a duplicate to line 30?

Reply:
Unfortunately classes inside the @media tag are less important. So it only works like this.